src/vsr/journal.zig

```diff
@@ -1643,7 +1643,7 @@ pub fn JournalType(comptime Replica: type, comptime Storage: type) type {
             assert(journal.has_dirty(message.header));
 
             const write = journal.writes.acquire() orelse {
-                assert(replica.replica_count > 1);
+                assert(!replica.sole_replica());
```

Comment:
Perhaps `replica.solo()` would read better?

Reply:
🤔 The part about *replica* is important semantically: sole_replica might still be a member of cluster with several standbys.

otoh, `solo` is quite catchy, and names the whole concept! I think on balance it’s better, will try rename